pkg/manager/impl/validation/validation.go

```diff
@@ -218,3 +218,12 @@ func ValidateLimit(limit uint32) error {
 	}
 	return nil
 }
+
+func ValidateProjectHasTag(project admin.Project, tag string) error {
+	if project.Labels != nil && project.Labels.Values != nil {
```

Comment:
if the tag is empty `""` then you will force users to have an empty lablel?

Reply:
No, fixing that.